dashboard/test/controllers/api/v1/pd/workshop_survey_report_controller_test.rb

```diff
@@ -101,20 +101,29 @@ class Api::V1::Pd::WorkshopSurveyReportControllerTest < ::ActionController::Test
     assert_equal Array.new(3, 'Jaime was very funny'), response_hash['this_workshop']['things_facilitator_did_well']
     assert_equal Array.new(3, 'Jaime was very funny'), response_hash['all_my_local_workshops']['things_facilitator_did_well']
 
-    [organizer, admin].each do |user|
-      sign_in(user)
-      @controller = @controller.class.new
-      get :local_workshop_survey_report, params: {workshop_id: workshop_2.id}
-      assert_response :success
-      response_hash = JSON.parse(@response.body)
-
-      assert_equal 1, response_hash['this_workshop']['how_much_learned']
-      assert_nil response_hash['all_my_local_workshops']['how_much_learned']
-      assert_equal({'Jaime' => 1.0}, response_hash['this_workshop']['how_clearly_presented'])
-      assert_nil response_hash['all_my_local_workshops']['how_clearly_presented']
-      assert_equal({'Jaime' => Array.new(3, 'Jaime was very funny')}, response_hash['this_workshop']['things_facilitator_did_well'])
-      assert_nil response_hash['all_my_local_workshops']['things_facilitator_did_well']
-    end
+    sign_in(organizer)
+    @controller = @controller.class.new
```

Comment:
So I did this because I wanted to build a large amount of sample data (that wasn't doable via a single factory) and then assert that that what is returned is different for different kinds of users.

I can always factor the data generation out to a private method for this.

It's not too _stinky_ to me, but who _nose_